### ISSUE TITLE
dependency on TERM variable

### ISSUE THREAD

**Issue by xsuchy** [rpm-software-management/tito#291]
Tito depends on existence of TERM variable. This is pitty as several CI teams use it in headless mode.

```
$ echo $TERM
xterm-256color
$ unset TERM
$ tito build --test --rpm  
Creating output directory: /tmp/tito
Traceback (most recent call last):
  File "/usr/bin/tito", line 23, in <module>
    CLI().main(sys.argv[1:])
  File "/usr/lib/python3.6/site-packages/tito/cli.py", line 202, in main
    return module.main(argv)
  File "/usr/lib/python3.6/site-packages/tito/cli.py", line 373, in main
    return builder.run(self.options)
  File "/usr/lib/python3.6/site-packages/tito/builder/main.py", line 142, in run
    info_out("Building package [%s]" % (self.build_tag))
  File "/usr/lib/python3.6/site-packages/tito/common.py", line 244, in info_out
    term = Terminal()
  File "/usr/lib/python3.6/site-packages/blessings/__init__.py", line 102, in __init__
    self._init_descriptor)
_curses.error: setupterm: could not find terminal
```

It is actually required by python3-blessings. But we should workaround it somehow so Tito can work even without this variable. Or we should it set in code to some default value when it is not defined.

**Comment by dgoodwin** [comment]
Agreed we should handle this gracefully.

**Comment by xsuchy** [comment]
https://github.com/erikrose/blessings/issues/39
But it is open for 4 years. :(

**Comment by ernestask** [comment]
Both blessings and blessed should have this fixed by now. python-blessed just needs to be updated in Fedora: https://bugzilla.redhat.com/show_bug.cgi?id=1792568

**Comment by FrostyX** [comment]
This shouldn't be blocked anymore as the `python3-blessed` update just arrived in Fedora stable repository.

**Comment by FrostyX** [comment]
Actually, seems like we didn't need to wait for

    Bug 1792568 - python-blessed-1.17.4 is available

This issue is fixed since 3a79fdd, even with python-blessed-1.16.1
